core/src/main/java/com/klaytn/caver/contract/Contract.java

```diff
@@ -205,6 +207,198 @@ public KlayLogs getPastEvent(String eventName, KlayLogFilter filterOption) throw
         return logs;
     }
 
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * @param methodName The smart contract method name to execute.
+     * @param methodArguments The arguments that need to execute smart contract method.
+     * @return List
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> call(String methodName, Object... methodArguments) throws NoSuchMethodException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
+        return call(CallObject.createCallObject(), methodName, methodArguments);
+    }
+
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * When creating CallObject, it need not to fill 'data', 'to' fields.
+     * The 'data', 'to' fields automatically filled in call() method.
+     * @param callObject A CallObject instance to 'call' smart contract method.
+     * @param methodName The smart contract method name to execute.
+     * @param methodArguments The arguments that need to execute smart contract method.
+     * @return List
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> call(CallObject callObject, String methodName, Object... methodArguments) throws NoSuchMethodException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
+        ContractMethod contractMethod = this.getMethod(methodName);
+        return contractMethod.call(Arrays.asList(methodArguments), callObject);
+    }
+
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * It is recommended to use this function when you want to execute one of the functions with the same number of parameters.
+     * @param methodName The smart contract method name to execute.
+     * @param methodArguments The arguments that need to execute smart contract method.
+     * @return List
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> callWithSolidityTypeReference(String methodName, Type... methodArguments) throws NoSuchMethodException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
```

Comment:
@sirano11 What about just use `callWithSolidityType`? 'type' and 'reference' together makes me confused.